index.html

```diff
@@ -1777,7 +1777,10 @@ <h2 id="Collection">Backbone.Collection</h2>
     <p id="Collection-remove">
       <b class="header">remove</b><code>collection.remove(models, [options])</code>
       <br />
-      Remove a model (or an array of models) from the collection, and returns them.
+      Remove a model (or an array of models) from the collection, and returns
+      them. <tt>models</tt> (or each of its elements if an array) can be any
```

Comment:
Personally I feel it's a little more confusing to refer to `get`. I have a slight pref to enumerate the options here.

Conversely, I'm not sure you need to spell out the "if models is an array" part twice. The plural kind of implies that.